src/main/java/org/embulk/input/jira/JiraInputPlugin.java

```diff
@@ -144,34 +159,38 @@ public TaskReport run(final TaskSource taskSource,
             }
             pageBuilder.finish();
         }
-        return Exec.newTaskReport();
+        return CONFIG_MAPPER_FACTORY.newTaskReport();
     }
 
     @Override
     public ConfigDiff guess(final ConfigSource config)
     {
         // Reset columns in case already have or missing on configuration
         config.set("columns", new ObjectMapper().createArrayNode());
-        final PluginTask task = config.loadConfig(PluginTask.class);
+        final PluginTask task = CONFIG_MAPPER.map(config, PluginTask.class);
         JiraUtil.validateTaskConfig(task);
         final JiraClient jiraClient = getJiraClient();
         jiraClient.checkUserCredentials(task);
         final List<Issue> issues = jiraClient.searchIssues(task, 0, GUESS_RECORDS_COUNT);
         if (issues.isEmpty()) {
             throw new ConfigException("Could not guess schema due to empty data set");
         }
+
+        // TODO: Eliminate the use of ExecInternal.getInjector that soon will be obsolete.
```

Comment:
Look likes this `TODO` is not resolved by current PR. Right?

Reply:
That's intentional, this PR only changed from `Exec.getInjector` to `ExecInternal.getInjector`. Later versions of Embulk won't allow `getInjector` at all.

Reply:
Ok got it